units/systemd-udevd.service.in

```diff
@@ -28,3 +28,4 @@ MountFlags=slave
 MemoryDenyWriteExecute=yes
 RestrictRealtime=yes
 RestrictAddressFamilies=AF_UNIX AF_NETLINK AF_INET AF_INET6
+SystemCallArchitectures=native
```

Comment:
Yeah, I think we should to the reasonable set, and I am pretty sure that mixing and matching architecture code in such a low-level part of the system is quite misguided.

That said, this should be documented in NEWS and downstream distros should be welcome to undo this change if they think it's necessary. It's pretty easy to do that after all.